src/textarea.rs

```diff
@@ -999,6 +813,15 @@ impl<'a> TextArea<'a> {
             false
         }
     }
+    fn normalize_selection(&self) -> ((usize, usize), (usize, usize)) {
+        let start = self.select_start.unwrap();
+        let end = self.cursor;
+        if start.0 > end.0 || start.1 > end.1 {
```

Comment:
This condition is not correct. For example consider `start = (1, 4)` and `end = (2, 3)`